src/error.rs

```diff
@@ -15,6 +15,9 @@ pub enum Error {
     /// Error that might occur when running on unsupported platforms.
     #[error("Unsupported platform.")]
     UnsupportedPlatformError,
+    /// Error that might occur when trying to show the cheat sheet
```

Comment:
Nit: can you end this comment with a "`.`" please?

Reply:
fixed it